### PR TITLE
Clean up from the removal of the client submodule

### PR DESCRIPTION
**Pull request by pivotal-mike**
- Clean up of makefile targets to finish removal of connectivity package.
- Adding a client folder + resources to support building clients and loaders

### CODE REVIEW

**Comment by hlinnaka** [comment]
We should not have an EULA, when the software is open source.

File: gpAux/client/install/src/unix/installer-header-template.txt [hunk 1]
```diff
@@ -0,0 +1,955 @@
...
+    TAIL="tail -n"
+else 
+    TAIL="tail"
+fi
+
+SKIP=`awk '/^__END_HEADER__/ {print NR + 1; exit 0; }' $0`
+
+more << EOF
+
+********************************************************************************
```
Comment (davecramer) [inline gpAux/client/install/src/unix/installer-header-template.txt]: What Heikki said ... EULA has no place here

Comment (pivotal-mike) [inline gpAux/client/install/src/unix/installer-header-template.txt]: Thanks you - I missed that during

**Comment by pivotal-mike** [comment]
@davecramer we need windows makefiles because we need to provide windows client/loader packages

**Comment by davecramer** [comment]
@pivotal-mike I thought that was the *only* reason we were buying clients?

**Comment by pivotal-mike** [comment]
@davecramer Clients == psql etc...  Loaders == gpfdist/gpload  Connectors == jdbc/odbc

**Comment by pivotal-mike** [comment]
@davecramer @hlinnaka Removed EULA - thank you for the catch

**Pull request merged**